modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -401,6 +408,10 @@ public Response getEpisodes(
       );
     }
 
+    if (StringUtils.isNotEmpty(live)) {
+      query.must(QueryBuilders.termQuery("live", live));
```

Comment:
```suggestion
      query.filter(QueryBuilders.termQuery("live", live));
```
This is simple binary logic, so we can use `filter` instead of `must`, which executes faster as no relevance score needs to be computed, and Elastic/OpenSearch can cache frequently used filters.